chapters/lighting/sections/environment/environment.tex

```diff
@@ -0,0 +1,66 @@
+\section{Environment}
+Even though the game can be played in non-Euclidean spaces which makes it inherently unrealistic, we decided to add some elements that would make the scenes portrayed in the game feel familiar.
```

Comment:
Technically our world is non-Euclidean and there are places where it is super hyperbolic so idk about the unrealistic

Reply:
Unrealistic in the sense that the world we live in is not Euclidean